product_main_supplier/views/product_template_views.xml

```diff
@@ -12,4 +12,48 @@
         </field>
     </record>
 
+    <!-- Should this view inherit and modify rather than replace? -->
```

Comment:
All these views were meant to simplify the view definitions which became a bit messy.

The fields `main_seller_id_price` and `main_seller_id_minimum_qty` should come in this module, the big view replacement can stay in `beesdoo_product_usability` for the time beeing.

Reply:
done